internal/attestation/crafter/crafter.go

```diff
@@ -186,7 +186,7 @@ func LoadSchema(pathOrURI string) (*schemaapi.CraftingSchema, error) {
 // Initialize the temporary file with the content of the schema
 func (c *Crafter) initCraftingStateFile(schema *schemaapi.CraftingSchema, wf *api.WorkflowMetadata, dryRun bool, runnerType schemaapi.CraftingSchema_Runner_RunnerType, jobURL string) error {
 	// Generate Crafting state
-	state, err := initialCraftingState(c.workingDir, schema, wf, dryRun, runnerType, jobURL)
+	state, err := c.initialCraftingState(c.workingDir, schema, wf, dryRun, runnerType, jobURL)
```

Comment:
Is it expected that the function belongs to the crafter, but also takes the crafter property as argument?
```
c.initialCraftingState(c.workingDir, ...)
```
Isn't workingDir already accessible without being an argument?

Reply:
Good catch, this is poor coding on my side. I'll fix it.